### DISCUSSION TITLE
Question on changing servers

### DISCUSSION THREAD

**Discussion by jpmiller25** [immich-app/immich#836] [Q&A] [answered]
If I want to change to a new server, do I need to do anything about backing up or moving the database?

Or do I just need to move the files in the upload folder and the database will recreate itself?

Thanks for the help,
Josh

**Comment by alextran1502** [comment, marked as answer]
Hello, you will need to move the docker managed volume `pgdata` and also the `UPLOAD_LOCATION` directory to your new server.
